src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 4;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        Color color = new ColorSupplier().getRandomColor();
```

Comment:
let's move the creating of ColorSupplier() object to the class level (as a `new Random()` object)